README.md

```diff
@@ -392,3 +392,24 @@ const onTodoChanged = (todo, index) => {
 ```
 
 Try to toggle some todos and restart the app, the changes should now be kept!
+
+# Adding new TODOs
+
+We need a way to add a TODO after the app started. We could navigate to a new page, add the TODO and go back, but for the sake of this exercise we'll use a (`Modal`)[https://facebook.github.io/react-native/docs/modal] instead.
+
+A `Modal` is a component that will be drawn over other widgets, and has 2 states: visible and invisible. Differently from native world, the modal component is always present in the tree, and controlling its visibility is done via the `visible` boolean property.
+
+Note that having a component in the tree in React doesn't necessarily mean that component will be present in the view hierarchy. React only generates a tree representing the intended view hierarchy, the native driver implementation that makes use of that data will then decide if to add or remove that view from the hierarchy.
```

Comment:
will then decide to add or remove (delete if)

Reply:
Maybe clarify this: What is the decision based on?

Reply:
6e81893